week1/ruby-1.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Rakshith Jain</title>
+        <style>
+            body{
+                background-color: lightgreen;
+                color:red;
+            }
+            header{
+                background-color: orange;
+            }
+            figure{
+                margin-left: 40%;
+
+            }
+            figcaption{
+                text-align: left;
+                color: darkblue;
+                font-style: italic;
+
+            }
+            table,tr,td,th{
+                border: 2px solid darkred;
+                margin-left: 45%;
+                font-size: x-large;
+            }
+            table{
+                border-collapse: collapse;
+            }
+            th{
+                font-weight: bolder;
+            }
+            h2{
+                margin-top: 4cm;
+                color: blueviolet;
+            }
+            footer{
+                text-align: right;
+                color: black;
+                font-weight: lighter;
+            }
+            section{
+                font-size: larger;
+            }
+
+
+
+
+        </style>
+    </head>
+    <body>
+        <!-- Header -->
+        <header>
+            <figure>
+            <img src="C:\Users\admin\Documents\photo.jpg" alt="My Pic">

```

Comment:
Unless the image is hosted elsewhere, always add relative paths to the image, so the code can work in any environment

Reply:
I have made the changes. Can u pls confirm?